6.2.2000_run.py

```diff
@@ -0,0 +1,14 @@
+import time
+
+
+def timer(f, *args, **kwargs):
+    st = time.time()
+    n = list(f(*args, **kwargs))
+    et = time.time()
+    elapsed_time = et - st
+    return elapsed_time
+
+
+print(timer(zip, [1, 2, 3], [4, 5, 6]))
```

Comment:
Use _name==main_ - 
https://docs.python.org/3/library/__main__.html
https://www.geeksforgeeks.org/python-main-function/amp/

Reply:
fixed it